### PR TITLE
[Support Us Page] [Populated] Help Our Team Section

### PR DESCRIPTION
**Pull request by daniel-panhead**
image

Yay cards

### CODE REVIEW

File: src/components/support-us/Card.tsx
Comment (IcedTea2K) [inline src/components/support-us/Card.tsx:1]: Is it intended to have white space between description and link for no-image cards on mobile?
image

**Review by IcedTea2K — CHANGES_REQUESTED** [review]
Very nice overall. Just a few small things on mobile version.

File: src/components/support-us/Card.tsx
Comment (daniel-panhead) [reply src/components/support-us/Card.tsx:1]: No :) Fixed now

**Review by IcedTea2K — APPROVED** [review]
LGTM!

**Pull request merged**